olcPGEX_AnimatedSprite.h

```diff
@@ -69,16 +69,17 @@ namespace olc
 		void SetState(std::string newState);
 		std::string GetState();
 		void Draw(float fElapsedTime, olc::vf2d position);
-		void AddState(std::string stateName, std::vector<std::string> imagePaths);
-		void AddState(std::string stateName, std::vector<olc::vi2d> spriteLocations);
+		void AddState(std::string stateName, float frameDuration, std::vector<std::string> imagePaths);
+		void AddState(std::string stateName, float frameDuration, std::vector<olc::vi2d> spriteLocations);
 		void SetSpriteSize(olc::vi2d size);
 		olc::vi2d GetSpriteSize();
 		void SetSpriteScale(float scale);
+		void SetFrameDuration(float duration);
```

Comment:
This isn't implemented anywhere?

Reply:
This is a vestige of something else I was playing with. Removed.